lib/ronin/db/web_vuln.rb

```diff
@@ -0,0 +1,195 @@
+      }
+
+      # @!attribute [rw] query_param
+      #   The query param of the URL.
+      #
+      #   @return [String, nil]
+      attribute :query_param, :string
+
+      # @!attribute [rw] header_name
+      #   The header name string part of the URL.
+      #
+      #   @return [String, nil]
+      attribute :header_name, :string
+
+      # @!attribute [rw] cookie_param
+      #   The cookie param of the URL.
+      #
+      #   @return [String, nil]
+      attribute :cookie_param, :string
+
+      # @!attribute [rw] form_param
+      #   The form param of the URL.
+      #
+      #   @return [String, nil]
+      attribute :form_param, :string
+      validate :param_validation
+
+      # @!attribute [rw] request_method
+      #   The request method for the URL.
+      #
+      #   @return [:copy, :delete, :get, :head, :lock, :mkcol, :move, :options, :patch, :post, :propfind, :proppatch, :put, :trace, :unlock]
+      enum request_method: {
+        copy:      'copy',
+        delete:    'delete',
+        get:       'get',
+        head:      'head',
+        lock:      'lock',
+        mkcol:     'mkcol',
+        move:      'move',
+        options:   'options',
+        patch:     'patch',
+        post:      'post',
+        propfind:  'propfind',
+        proppatch: 'proppatch',
+        put:       'put',
+        trace:     'trace',
+        unlock:    'unlock'
+      }, _suffix: :requests
+
+      # @!attribute [rw] lfi_os
+      #   The lfi os.
+      #
+      #   @return [:unix, :windows, nil]
+      enum lfi_os: {
+        unix:    'unix',
+        windows: 'windows'
+      }
+
+      # @!attribute [rw] lfi_depth
+      #   The lfi depth.
+      #
+      #   @return [Integer, nil]
+      attribute :lfi_depth, :integer
+
+      # @!attribute [rw] lfi_filter_bypass
+      #   The lfi filter bypass.
+      #
+      #   @return [:null_byte, :base64, :rot13, :zlib, nil]
+      enum lfi_filter_bypass: {
+        null_byte: 'null_byte',
+        base64:    'base64',
+        rot13:     'rot13',
+        zlib:      'zlib'
+      }, _prefix: true
+
+      # @!attribute [rw] rfi_script_lang
+      #   The rfi script lang.
+      #
+      #   @return [:asp, :asp_net, :cold_fusion, :jsp, :php, :perl, nil]
+      enum rfi_script_lang: {
+        asp:         'asp',
+        asp_net:     'asp_net',
+        cold_fusion: 'cold_fusion',
+        jsp:         'jsp',
+        php:         'php',
+        perl:        'perl'
+      }
+
+      # @!attribute [rw] rfi_filter_bypass
+      #   The rfi filter bypass.
+      #
+      #   @return [:null_byte, :double_encode, nil]
+      enum rfi_filter_bypass: {
+        null_byte: 'null_byte',
+        double_encode: 'double_encode'
+      }, _prefix: true
+
+      # @!attribute [rw] rfi_filter_bypass
+      #   The rfi filter bypass.
+      #
+      #   @return [:null_byte, :double_encode, nil]
+      enum ssti_escape: [:foo, :erb] # ! https://github.com/ronin-rb/ronin-vulns/issues/55
```

Comment:
Rename to `ssti_escape_type` and copy/paste the values from: https://github.com/ronin-rb/ronin-vulns/blob/842bffaaf10d1c427002e826c7d825c9275f11a9/lib/ronin/vulns/ssti.rb#L46